content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Applying Pod Security Standards at Cluster level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Note" %}}
+This tutorial applies only for new clusters.
+{{% /alert %}}
+
+Since v1.23, Pod Security admission (PSA) is enabled by default as it [graduated
+to beta](blog/_posts/2021-12-15-pod-security-admission-beta.md). In this 
+tutorial, we will enforce `baseline` Pod Security Standard, at a cluster level.
```

Comment:
@jimangel / @lachie83 can you take a look at the call out for the blog post here and add your comments / suggestions?

Reply:
```suggestion
Since v1.23, Pod Security Admission (PSA) is enabled by default as it [graduated
to beta](blog/_posts/2021-12-15-pod-security-admission-beta.md). Pod Security Admission
is an admission controller that applies Pod Security Standards when pods are created.

This tutorial shows you how to enforce the `baseline` Pod Security Standard at the cluster level.
```

Reply:
One more sentence about the difference between cluster and namespace level. Is it that all namespaces get the same treatment? Link from here to the Namespace tutorial so users can immediately jump to it too.

Reply:
done